src/components/SidePanel/HelpComponents/HelpContent.tsx

```diff
@@ -90,13 +90,17 @@ function HelpContent({closeSidePanel}: HelpContentProps) {
                 shouldShowCloseButton={isExtraLargeScreenWidth}
                 shouldDisplayHelpButton={false}
             />
-            <ScrollView
-                style={[styles.ph5, styles.pb5]}
-                userSelect="auto"
-                scrollIndicatorInsets={{right: Number.MIN_VALUE}}
-            >
-                {getHelpContent(styles, route, isProduction, expandedIndex, setExpandedIndex)}
-            </ScrollView>
+            {currentState === undefined ? (
+                <FullScreenLoadingIndicator />
```

Comment:
```suggestion
                <FullScreenLoadingIndicator style={[styles.flex1, styles.pRelative]} />
```
Let's update `FullScreenLoadingIndicator` style so it won't overlap the header